src/main/java/hillel/spring/service/HelloService.java

```diff
@@ -0,0 +1,48 @@
+package hillel.spring.service;
+
+import hillel.spring.database.model.HelloModel;
+import hillel.spring.repository.HelloRepository;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Random;
+
+@Service
+public class HelloService {
+
+    public HelloRepository getHelloRepository() {
+        return helloRepository;
+    }
+
+    private final HelloRepository helloRepository;
+    private final Random random = new Random();
+
+    public HelloService(HelloRepository helloRepository) {
+
+        this.helloRepository = helloRepository;
+    }
+
+    public String getHello(String language) {
+        return helloRepository.getDatabase().getLanguagesHello().stream()
+                .filter(model -> model.getLanguageName().trim().equals(language))
+                .findFirst()
+                .map(HelloModel::getGreeting)
+                .orElse("No Such Language, Sorry");
+    }
+
+    public String getRandomHello() {
+
+       int index = random.nextInt(helloRepository.getDatabase().getLanguagesHello().toArray().length);
+       HelloModel helloModel = (HelloModel) helloRepository.getDatabase().getLanguagesHello().toArray()[index];
```

Comment:
Rule of thumb is to use a `List` and an `ArrayList` as defaults (in other words - everywhere). 
You have chosen a `Set` (without any good reason for that) and here you are struggling with that choice, converting `Set` to `Array`.
So I would recommend using `List` instead, and in this method you will have index-based access out of the box.